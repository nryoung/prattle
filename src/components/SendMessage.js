import React from 'react';
import styled from 'styled-components';

const StyledSendMessage = styled.div`
  position: fixed;
  bottom: 0;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 56px;
`;

const SendMessage = () => (
  <StyledSendMessage>send message here</StyledSendMessage>
);

export default SendMessage;
