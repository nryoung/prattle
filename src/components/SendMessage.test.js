import React from 'react';
import renderer from 'react-test-renderer';
import SendMessage from './SendMessage';

it('it renders', () => {
  const component = renderer.create(<SendMessage />);
  expect(component.toJSON()).toMatchSnapshot();
});
