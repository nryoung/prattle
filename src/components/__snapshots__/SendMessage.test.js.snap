// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="sc-bdVaJa ctXHea"
>
  <form
    className="sc-bwzfXH lejAfz"
    onSubmit={[Function]}
  >
    <input
      className="sc-htpNat bdmXgv"
      onChange={[Function]}
      placeholder="Type your message here and hit ENTER!"
      type="text"
      value=""
    />
  </form>
</div>
`;
